scripts/test_and_import_gtfs.sh

```diff
@@ -15,6 +15,7 @@ export PGUSER=$(sed -r "s/$DATABASE_REGEX/\1/" <<< "$DATABASE_URL")
 export PGPASSWORD=$(sed -r "s/$DATABASE_REGEX/\2/" <<< "$DATABASE_URL")
 export PGHOST=$(sed -r "s/$DATABASE_REGEX/\3/" <<< "$DATABASE_URL")
 export PGDATABASE=$(sed -r "s/$DATABASE_REGEX/\4/" <<< "$DATABASE_URL")
+export PGSSLMODE=require
```

Comment:
Instead of exporting this, which might interfere with the already set `PGSSLMODE` in our container, I'd run the commands that require this variable (`psql`, `make`) by passing this variable to them, i.e. `PGSSLMODE=require psql ...` and `PGSSLMODE=require make ...`.